src/components/Table/GridTable.stories.tsx

```diff
@@ -1136,3 +1136,99 @@ export function ToggleCustomCollapse() {
     </>
   );
 }
+
+type ExpandHeader = { id: "expandableHeader"; kind: "expandableHeader" };
+type Header = { id: "header"; kind: "header" };
```

Comment:
I think we do export the `Header` as `SimpleHeader`. We could also do a `SimpleExpandHeader` as well. "Why `Simple...`?", you ask? Because sometimes we may want to include some data in the row types, like:
```
type Header = { id: "header", kind: "header", data: SomeFragment }
```
Then we can use the data in the row to help render the header.